{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='192.168.1.5', port=8080): Max retries exceeded with url: /wd/hub/session (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa99c1f5710>, 'Connection to 192.168.1.5 timed out. (connect timeout=None)'))","children":[{"name":"test_for_authorization","uid":"5aa0b7b8da65f965","parentUid":"0844c5ae2f86ff87c155ae7980381edc","status":"broken","time":{"start":1691317417466,"stop":1691317417466,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"0844c5ae2f86ff87c155ae7980381edc"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}